tnt/v1/tnt.yaml

```diff
@@ -0,0 +1,299 @@
+openapi: 3.0.0
+info:
+  version: "1.1.0"
```

Comment:
Are we still fixing 1.1?

Reply:
I guess this is just a copy of the old... Difficult to see if anything has been changed...